csrc/runtime/executor_kernel_arg.cpp

```diff
@@ -61,30 +61,30 @@ void KernelArgumentHolder::push(const std::vector<at::Tensor>& tensors) {
   }
 }
 
-void KernelArgumentHolder::erase(const PolymorphicValue* arg_to_delete) {
+void KernelArgumentHolder::erase(const PolymorphicValue& arg_to_delete) {
   auto iter = std::remove_if(
       arguments_.begin(), arguments_.end(), [&](const auto& ref) {
-        return arg_to_delete == ref.get();
+        return arg_to_delete == ref;
```

Comment:
Should this be `&arg_to_delete == &ref`?

Reply:
Good catch, thank you!